environs/config/config.go

```diff
@@ -1839,11 +1877,26 @@ global or per instance security groups.`,
 		Type:        environschema.Tattrs,
 		Group:       environschema.EnvironGroup,
 	},
-	"rsyslog-ca-cert": {
+	RsyslogURL: {
```

Comment:
i agree.. i expect there will be a doc, when the whole rsyslog forwarding feature is implemented and that would be a good place to document this behaviour